app/src/pipeline3/host-preparation.js

```diff
@@ -68,6 +68,23 @@ if (process.env[config.varDeploymentChanges()] === "true") {
   }
 }
 
+// 'data'
+
+if (process.env[config.varDataChanges()] === "true") {
+  for (var index in instanceDef.data) {
+    var hostDataDir = hostDir + "/data";
+    var data = instanceDef.data[index];
+    if (data.type === "instance") {
+      // Retrieve the data directory of the other instance
+      var hostSourceDataDir =
```

Comment:
I think this is cleared:
* `hostSourceDataDir` ➔ `sourceInstanceDataDir`
* `hostDataDir ` ➔ `instanceDataDir`

Also
>`// Retrieve the data directory of the source instance`